src/main/java/com/ibreath/service/UserService.java

```diff
@@ -0,0 +1,88 @@
+package com.ibreath.service;
+
+import com.ibreath.enumeration.GenderEnum;
+import com.ibreath.resource.dto.UserDto;
+import com.ibreath.resource.model.entity.UserEntity;
+import com.ibreath.resource.model.repository.UserRepository;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.security.crypto.bcrypt.BCryptPasswordEncoder;
+import org.springframework.security.crypto.password.PasswordEncoder;
+import org.springframework.stereotype.Service;
+
+import org.apache.log4j.Logger;
+
+import javax.transaction.Transactional;
+
+@Service
+@Transactional
+public class UserService {
+
+    private UserRepository repository;
+
+    private static Logger logger = Logger.getLogger(UserService.class);
+
+    public UserDto getUser(String userId) throws Exception {
+
+        try {
+            UserEntity entity = repository.findById(Long.valueOf(userId));
+            UserDto dto = new UserDto();
+            dto.setFirstname(entity.getFirstname());
+            dto.setLastname(entity.getLastname());
+            dto.setEmail(entity.getEmail());
+            dto.setBirthDate(entity.getBirthDate());
+            dto.setWeight(entity.getWeight());
+            dto.setHeight(entity.getHeight());
+            dto.setDecreaseRate(entity.getDecreaseRate());
+            dto.setGender(entity.getGender());
+            return dto;
+        } catch (Exception e) {
+            logger.fatal("Error with finding userId in getUser method : " + e.getMessage());
+            throw new Exception("The user doesen't exist");
+        }
+    }
```

Comment:
Why do you need a try catch ? you can just test your first result of the repository like 
if (entity == null) 
  Throw new etc...